Tests/Functional/MissingAllTestsOverride/main.swift

```diff
@@ -0,0 +1,22 @@
+// RUN: %{swiftc} %s -o %{built_tests_dir}/MissingAllTestsOverride
+// RUN: %{built_tests_dir}/MissingAllTestsOverride > %t || true
+// RUN: %{xctest_checker} %t %s
+// CHECK: Test Case 'MissingAllTestsOverride.missingAllTestsOverride' started.
+// CHECK: <EXPR>:0: unexpected error: MissingAllTestsOverride.missingAllTestsOverride : threw error "This test case class must override the allTests property" - 
```

Comment:
I don't *think* that is possible, at least not without calling out to some sort of symbolication utility.

Currently this is using the unhandled-error functionality that @eschaton added a while back, producing the same sort of output as here: https://github.com/apple/swift-corelibs-xctest/blob/master/Tests/Functional/ErrorHandling/main.swift#L13